networking/configure-syscontrols-interface-tuning-cni.adoc

```diff
@@ -0,0 +1,23 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="configure-syscontrols-interface-tuning-cni"]
+= Configuring system controls and interface attributes using the tuning plugin
+include::_attributes/common-attributes.adoc[]
+:context: configure-syscontrols-interface-tuning-cni
+
+toc::[]
+
+In {product-title} you can configure the tuning CNI plugin to change some sysctls and several interface attributes (promiscuous mode, all-multicast mode, MTU, and MAC address) in the network namespace. The tuning CNI meta plugin operates in a chain with a main CNI plugin as illustrated. The main CNI plugin assigns the interface and passes this to the tuning CNI meta plugin at runtime.
```

Comment:
this is different to what we have in the current documentation. Why?
https://docs.openshift.com/container-platform/4.14/networking/setting-interface-level-network-sysctls.html

Reply:
Yes I agree but if you read that section the focus really seemed to be on sysctls I wanted to rewrite it but also reflect that you can do other things with the tuning CNI like modify the multicast mode. I was reading again it does seem to make sense, but I will look at again. Previously it was mostly about changing sysctls I wanted to reflect that it can do more. Begs the question, do we also support changing promiscuous mode, the MTU, and MAC addresses ?

Reply:
What I'm trying to say is that the current document also talks about changing other interface attributes. So instead of modifying I would suggest to simply change the title and remove the note that says that only sysctl are supported.

Changing promiscuous mode, the mtu and mac address also works but only allmulti will be officially supported since it is tested. However, I don't think we should mention that, we don't want users to believe that those things doesn't work on OCP.